api_pagopa.yaml

```diff
@@ -0,0 +1,289 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy PagoPA
+  description: Proxy for PagoPA payment services
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  "/payment/check":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: checkPayment
+      summary: Check Payment
+      description: Retrieve information about a payment
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: "#/definitions/PaymentsCheckRequest"
+          x-examples:
+            application/json:
+              codiceIdRPT:
+                CF: "TMMEXQ60A10Y526X"
+                AuxDigit: 1
+                CodIUV: 123456789012345
+                CodStazPA: 56
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            "$ref": "#/definitions/PaymentsCheckResponse"
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: "ABC123"
+        "400":
+          description: Input provided or pagoPA response are invalid
+        "500":
+          description: PagoPA services are not available or request is rejected
+  "/payment/activation":
+    x-swagger-router-controller: PaymentController
+    get:
```

Comment:
`POST /payment/{CF}` (or IMHO better, `PUT /payment/{CF}/{codiceAvviso}`)

Reply:
definitely NOT a GET as this is not a _safe_ operation - @LsKoder I suggest to review the [properties of HTTP methods](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods)

Reply:
Yep, it was just a mistake defining it into yaml file. Really it's a post (into App.ts). I fixed it.